extensions/3DTILES_content_voxels/README.md

```diff
@@ -201,4 +211,48 @@ Tileset JSON with implicit tiling
+
+The following notes explain the implications for **box** and **cylinder** voxel shapes, including ways that runtimes can incorporate the `y`-up to `z`-up transform as they render voxel data.
+
+#### Box
+
+For any given glTF model, a `y`-up to `z`-up rotation will rotate its `y` axis to the positive `+z` axis, and its `z` axis to the negative `-y` axis. In a box voxel grid, the rotation transforms the indices of the voxel data in a similar way.
+
+Given a box grid with dimensions `[width, height, depth]`, a voxel at index `[x, y, z]` in the glTF will be relocated to `[x, z, (height - 1) - y]` in 3D Tiles (see figure below).
+
+![](figures/box-voxel-transform.png)
+
+In a UV-sampling approach, the original UV coordinates would be converted from `[u, v, w]` to `[u, w, 1.0 - v]`.
+
+Conversely, if the dimensions are `[width, height, depth]` in 3D Tiles, a box voxel that is located at `[x, y, z]` will relocate to `[x, (depth - 1) - z, y]` in the glTF. Their UV coordinates respectively would be converted from `[u, v, w]` to `[u, 1.0 - w, v]`.
+
+#### Cylinder
+
+`3DTILES_bounding_cylinder` is modeled closely after the `EXT_implicit_cylinder_region` extension in glTF. However, the start of the cylinder's angle is different between the two coordinate systems. In 3D Tiles, an angle of `0` aligns with the positive `+x` axis, whereas in glTF it aligns with the negative `-x` axis. The resulting difference of `pi` should be accounted for when rendering the data.
+
+As long as the region's angular range is less than `2pi`, the ordering of the voxel data remains the same. Instead, the difference can be treated as an offset in the `minAngle` and `maxAngle` properties of the region.
+
+Given an `EXT_implicit_cylinder_region` in glTF, its equivalent in `3DTILES_bounding_cylinder` can be deduced by adding `pi` to its angular bounds. If the resulting angle is greater than `2pi`, then it should be wrapped around to its equivalent in the `[-pi, pi]` range (see below).
+
+![](figures/cylinder-angle-rotation.png)
+
+No matter how the `minAngle` and `maxAngle` change, a voxel located at `[x, y, z]` in the glTF will still be located at [`x, z, y`] in 3D Tiles, and vice versa. (`y` and `z` are swapped due to the different up-axes.)
+
+However, this rule does not work if the cylinder region fills the entire `2pi` range (such that `minAngle = -pi` and `maxAngle = pi`). In this case, the indices used to access the voxel data will be affected. A **cylinder** voxel at `[x, y, z]` within the glTF will be located at `[x, z, a]` in 3D Tiles, where `a = [y + (angle / 2)] % angle`. The math is the same vice versa.
```

Comment:
Are `y` and `z` switched here?

A cylinder voxel at `[x, y, z]` within the glTF has its angle defined by its `z` coordinate, and its height defined by `y`. So the 3D Tiles coordinate would be `[x, a, y]`, where `a = (y + angleRange / 2) % angleRange`.

I was initially confused by the word `angle`, until I realized it meant the tile dimension along the angle axis. Would `angleRange` be clearer?

Reply:
Ah you're right, thanks for the catch @jjhembd !